docs/download.md

```diff
@@ -31,7 +97,7 @@ Software Requirements:
 
 	CUDA & NVIDIA Drivers*: 11.0-11.4 & v450.80.02+
 
-	Apache Spark 3.0.1, 3.0.2, 3.0.3, 3.1.1, 3.1.2, 3.2.0, Cloudera CDP 7.1.6, 7.1.7, Databricks 7.3 ML LTS or 8.2 ML Runtime, and GCP Dataproc 2.0
+	Apache Spark 3.0.1, 3.0.2, 3.0.3, 3.1.1, 3.1.2, 3.2.0, Cloudera CDP 7.1.6, 7.1.7, Databricks 7.3 ML LTS or 8.2 ML Runtime, GCP Dataproc 2.0, and Azure Synapse
```

Comment:
we put Azure Synapse here, but they are in progress of upgrading did we want to put something for Azure and EMR that say see their pages for current version?